src/Tools/LeanCode.CodeAnalysis/NetStandard21Compatibility/MissingFileMethods.cs

```diff
@@ -0,0 +1,42 @@
+// Based on https://github.com/dotnet/runtime/blob/5535e31a712343a63f5d7d796cd874e563e5ac14/src/libraries/System.Private.CoreLib/src/System/IO/File.cs
+// as linked by docs on 02.09.2024
+
+using System.Text;
+
+namespace System.IO;
+
+public static class MissingFileMethods
```

Comment:
One alternative to consider for this would be just

```suggestion
namespace LeanCode.CodeAnalysis;

public static class File
{
    public static void Delete(string path) => global::System.IO.File.Delete(path);
```

which would get picked up instead of `System.IO.Path` where appropriate and avoid the need to call methods here by current (quite awkward) class name.

Reply:
If I put it in `LeanCode.CodeAnalysis` namespace it gets picked up in `LeanCode.CodeAnalysis.Tests`, which make use of `File.ReadAllTextAsync`, so I would have to also implement that or add a `using` there. I think that it currently works, because tests still target .NET8.

Reply:
What do you think of a global `using` alias in `LeanCode.CodeAnalysis`?